applications/crowdloan_frontend_template.md

```diff
@@ -0,0 +1,189 @@
+| 0a. | License | MIT or TBD                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| 0b. | Documentation | For both approaches (Webflow & Jamstack) we provide how-to guides. Jamstack will be documented in the Github repo.                                                                                                                                                                                                                                                                                                                                           |
+| 0c. | Article | We will publish an article presenting the templates and how to use them.                                                                                                                                                                                                                                                                                                                                                                                     |
+| 1a. | Benchmarking | Design research of existing Crowdloan pages and other one pages designs, like Github project pages templates. <br>This will allow us to synthesize viable options of the page designs                                                                                                                                                                                                                                                                        |
+| 1b. | Design Direction Prototype | Aiming to create as many medium to high-fidelity dd prototypes as possible that allows the Grants team and the community to have an input on the design direction<br>The aim is to have the prototypes at 25%-50% completeness, to see major components/features and the general design direction. This way we don't waste time on dismissed design directions.<br>The designs should follow good practices in general without requiring additional research |
+| 1c. | Repo Setup | Repo setup incl. base libraries/frameworks, initial technical documentation. Undesigned base scaffold. Allows the implementation to be simplified by forking                                                                                                                                                                                                                                                                                                 |
```

Comment:
> Undesigned base scaffold

Does that mean, that with 1c the whole code for the SSG is already finished, including all the structure and logic, but not the theme? And in 2) and 3) you'd create an example theme?!

Reply:
This would be the project structure with all necessary dependencies as far as the engineer knows them at the start. It will only have the content structure as per brief, without any recommendations/feedback that come during the design process. As for logic, we assume the engineer will start the blockchain integration at this point as it has some design independent parts. Whether or not they will finish it remains to be seen

Reply:
Thanks. But I'm still not clear if what you meant by

> One (1) chosen design direction

refers to _visual design_ or _software design/architecture_.
If visual design, my assumption would be that 2) and 3) are about creating an example theme, which could be reused or replaced by a specific theme by anybody who is going to use your template. Could you clarify this for me?

> As for logic, we assume the engineer will start the blockchain integration at this point as it has some design independent parts.

Also, to which deliverable do you refer by "at this point"?

Reply:
- These are indeed going to be visual templates. One in Gatsby(react) another in Webflow. They will be white-label in the sense that the design and functionality allow for the app to function. And engineers can re-skin/re-theme them as they wish.

- When referring to "at this point" we are talking about the point at which an engineer is marrying the UI to the actual blockchain logic/functionality

Reply:
Ok sounds good, thanks for clearifying @adrianloffredo